types/model/file.go

```diff
@@ -10,11 +10,45 @@ import (
 	"strings"
 )
 
+type File struct {
+	Commands []Command
```

Comment:
It would be so much more ergonomic to have `From string, Adapter string, Messages []Message, Params []Param...` vs asking the user to loop over all of the things looking for the one thing they are looking for.

Reply:
See https://github.com/ollama/ollama/blob/bmizerany/x/x/model/file.go#L102

Reply:
The original reason this was implemented as commands was to maintain some order in the result. It didn't turn out to be very useful so I might make this change in a follow up. 

Also this struct is almost identical to the Model struct so it might be a good candidate for some clean up